config/settings.yml

```diff
@@ -43,6 +43,8 @@ replication:
   audit_should_backfill: false
 
 total_worker_count: 117 # for okcomputer endpoint
+minimum_subfolder_count: null # for okcomputer pres-cat mount check, verifies the storage_trunk has at least this many subfolders
```

Comment:
what about a default of 1 in the base settings (and then we can increase the threshold in shared_configs for qa/stage/prod after this is merged)?  i believe all 3 deployed envs can be expected to have at least some Moabs, so a non-null threshold seems preferable to me, at first glance.

Reply:
sure, set to 1

Reply:
thanks!